buffer/src/test/java/io/netty/buffer/ReadOnlyDirectByteBufferBufTest.java

```diff
@@ -236,6 +236,20 @@ public void testGetReadLong() {
         buf.release();
     }
 
+    @Test(expected = IndexOutOfBoundsException.class)
+    public void testGetBytesByteBuffer() {
+        byte[] bytes = {'a', 'b', 'c', 'd', 'e', 'f', 'g'};
+        // Ensure destination buffer is bigger then what is in the ByteBuf.
+        ByteBuffer nioBuffer = ByteBuffer.allocate(bytes.length + 1);
```

Comment:
its kind of wierd to allocate a non-direct buffer in a class that is named direct.

Reply:
@carl-mastrangelo I'm not sure it matters since it's just a separate destination to read bytes into, the fact that the source is a direct buffer I think is kind of unrelated?